src/clims/services/extensible.py

```diff
@@ -220,25 +222,27 @@ def __init__(self, **kwargs):
             # Stop here as we're wrapping a version that already exists
             return
 
-        name = kwargs.pop("name", None)
-        org = kwargs.pop("organization", None)
-
-        if not name or not org:
-            raise AttributeError("You must supply name and organization")
+        if not name:
+            raise AttributeError("You must supply a name")
 
         extensible_type = self._app.extensibles.get_extensible_type(self.type_full_name)
         self._archetype = self.WrappedArchetype(name=name, extensible_type=extensible_type,
-                organization=org)
+                organization=self._context.organization)
         self._wrapped_version = self.WrappedVersion()
 
         # Add any remaining properties in kwargs. This is necessary so that user
         # can instantiate objects using e.g. syntax like: Sample(my_value=1)
         for key, value in six.iteritems(kwargs):
+            assert key != "organization"
```

Comment:
Is this a debug thing that hasn't been removed?

Reply:
Yes! I was going to remove it after I found all the organizations.